KsApi/models/graphql/GraphBackingTests.swift

```diff
@@ -63,6 +74,17 @@ final class GraphBackingTests: XCTestCase {
                 "imageUrl": "http://www.kickstarter.com/avatar.jpg",
                 "id": "VXNlci0xMjA3OTk3NjQ5",
                 "name": "Creator McBaggins",
+                "storedCards": [
+                  "nodes": [
+                    [
+                      "expirationDate": "2023-01-01",
+                      "id": "6",
+                      "lastFour": "4242",
+                      "type": "VISA"
+                    ]
```

Comment:
It might be simpler and less json to return empty arrays of objects we don't directly test. In this case we technically don't even check the stored cards, just that the JSON builds with some (even an empty) array of stored cards.

Reply:
It's fine in this case, I just mean when we have like 8 objects in an array, we don't absolutely need to make every json file that reads in those objects the same.

Reply:
Yeah good point. The reason you see a lot of copy pasted json is because the UserFragment is referenced in a lot of different places. When applying this migration it was breaking everywhere so I was individually debugging a ton of places. Will keep this in mind moving forward.